library/SimplePie/Item.php

```diff
@@ -90,7 +90,26 @@ class SimplePie_Item
 	public function __construct($feed, $data)
 	{
 		$this->feed = $feed;
-		$this->data = $data;
+		$this->data = $this->array_map_recursive('trim', $data);
```

Comment:
@mblaney The space you we're talking about, it's non-existing because of this line...